docs/flip-flop_conversions.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: default
+title: Flip-Flop Conversions
+comments: true
+nav_order: 14
+---
+
+# Flip-Flop Conversions
+{: .no_toc }
+
+This page will teach you how to convert one type of Flip flop to another type.
+
+<h2>Steps to be followed :</h2>
+ <ul>
+   <li> Identify available and required Flip flop. </li>
+   <li> Make characteristic table for required Flip flop. </li>
+   <li> Make excitation table for available Flip flop. </li>
+   <li> Write boolean expression for available Flip flop. </li>
+   <li> Draw the circuit. </li>
+ </ul>
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+
+
+## JK Flip-Flop to D Flip-Flop
+
+We can convert JK Flip-Flop to D Flip-Flop by following the above mentioned steps.
+
+<h3>Step 1</h3>
```

Comment:
Use Markdown syntax:
```### Step 1```

Reply:
when I do like that, it gets added to the table of contents

Reply:
Ok. I don't think having it in the table of contents is a bad thing, but if you don't want it there just leave those types of headings as `<h3>`